### PR TITLE
Ignore dotenv in dart

### PR DESCRIPTION
**Pull request by Manoj-Paramsetti**
# Pull Request

### Update

- [ ] Template - Update existing `Dart.gitignore` template

## Details

**Reasons for making this change:**
Ignoring dotenv files to avoid leaking secrets on Github

**Links to documentation supporting these rule changes:**
Reason for ignoring the dotenv file on GitHub
https://blog.gitguardian.com/secrets-api-management/#git-ignore
dotenv package on pub.dev
https://pub.dev/packages/flutter_dotenv

## Related pull request in Github/gitignore - (Created by me)
[#3709-Github](https://github.com/github/gitignore/pull/3709)

### CODE REVIEW

**Review by filipechagas — COMMENTED** [review]
Thank you for contributing @Manoj-Paramsetti
I've left a comment for your appreciation.
Also, please use a patch file instead of changing main `.gitignore` one. Refer to our documentation for more information: https://github.com/toptal/gitignore#files

File: templates/Dart.gitignore [hunk 1]
```diff
@@ -11,6 +11,10 @@ pubspec.lock
 # If you don't generate documentation locally you can remove this line.
 doc/api/
 
+# dotenv environment variables file
+.env
+.env.test
```
[the remark this answers is not in the document]
Comment (Manoj-Paramsetti) [reply templates/Dart.gitignore:16]: some people use `.env.example` to show their environment file structure so I used `.env` and `.env.test` separately

**Comment by Manoj-Paramsetti** [comment]
> Thank you for contributing @Manoj-Paramsetti
> I've left a comment for your appreciation.
> Also, please use a patch file instead of changing main `.gitignore` one. Refer to our documentation for more information: https://github.com/toptal/gitignore#files

Okay, I will make those changes

File: templates/Dart.gitignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (Manoj-Paramsetti) [reply templates/Dart.gitignore:16]: @filipechagas I saw that the changes you requested is already exist in this repository

[the remark this answers is not in the document]
Comment (Manoj-Paramsetti) [reply templates/Dart.gitignore:16]: I think ignoring only `.env` is enough for environment file

**Review by filipechagas — CHANGES_REQUESTED** [review]
@Manoj-Paramsetti Can you please make it a `.patch` instead of `.gitignore`. 
This will also fix merging conflicts.

**Comment by Manoj-Paramsetti** [comment]
> @Manoj-Paramsetti Can you please make it a `.patch` instead of `.gitignore`.
> This will also fix merging conflicts.

@filipechagas Updated my commit. You can check now

**Review by deniskaber — APPROVED** [review]

**Pull request merged**